src/dk.nita.saml20/dk.nita.test.saml20/Saml20/Schema/BasicPrivilegeProfile/PrivilegeListTypeTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using dk.nita.saml20.Schema.BasicPrivilegeProfile;
+using dk.nita.saml20.Utils;
+using NUnit.Framework;
+
+namespace dk.nita.test.Saml20.Schema.BasicPrivilegeProfile
+{
+    [TestFixture]
+    public class PrivilegeListTypeTest
+    {
+        [Test]
+        public void CanDeserializeXml()
```

Comment:
Maybe add a negative test as well verifying that the right url must be specified!